util/ci.sh

```diff
@@ -12,10 +12,6 @@ case $1 in
   before_script)
     if [ "$TEST_TOOL" = "rubygems" ]
     then
-      gemhome=$(gem env home)
-
-      ruby -I lib bin/gem uninstall executable-hooks gem-wrappers bundler-unload -x --force -i "$gemhome@global"
-
```

Comment:
Should we move this to the rvm workflow, or is it not necessary at all? I tend to prefer to actually remove it, because a ruby installation with these tweaks is actually what rvm users get, so it feels best to test against more real world installations :+1:.